packages/@remirror/core/src/builtins/__tests__/persistent-selection-extension.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { extensionValidityTest, renderEditor } from 'jest-remirror';
+
+import { PersistentSelectionExtension } from '..';
+
+extensionValidityTest(PersistentSelectionExtension);
+
+describe('configuration', () => {
+  [
+    {},
+    { persistentSelectionClass: undefined },
+    { persistentSelectionClass: '' },
+    { persistentSelectionClass: 'selection' },
+  ].forEach((configuration) => {
+    test(`uses provided configuration (${JSON.stringify(configuration)}`, () => {
+      const {
+        add,
+        dom,
+        selectText,
+        nodes: { p, doc },
+      } = renderEditor([], { builtin: configuration });
+
+      add(doc(p('Some text')));
+      selectText({ from: 4, to: 9 });
+
+      expect(dom.outerHTML).toMatchSnapshot();
+    });
+  });
+});
+
+describe('focus', () => {
+  test('retains selection and decoration', () => {
```

Comment:
Nitpick, I prefer `test()` when it's at top level and `it()` when nested in a `describe` block. 

I don't even know why, and you can leave it as is.